activiti-steps/activiti-steps-assertions/src/main/java/org/activiti/steps/LocalTaskProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps;
+
+import java.util.List;
+
+import org.activiti.api.runtime.shared.query.Page;
+import org.activiti.api.runtime.shared.query.Pageable;
+import org.activiti.api.task.model.Task;
+import org.activiti.api.task.model.builders.TaskPayloadBuilder;
+import org.activiti.api.task.runtime.TaskRuntime;
+
+public class LocalTaskProvider implements TaskProvider {
```

Comment:
@erdemedeiros related to my previous comments I wonder if this can be called InMemoryTaskSource (it might be confusing due the tasks are in the DB) for the future iterations.

Reply:
Yeah, InMemoryTaskSource is confusing because tasks are in DB, but maybe `TaskRuntimeTaskSource`? or keep `LocalTaskSource`?

Reply:
keep LocalTaskSource for now